challenges/tree/README.md

```diff
@@ -10,6 +10,7 @@
 - Create a BinaryTree class
   - Define a method for each of the depth first traversals called `preOrder`, `inOrder`, and `postOrder` which returns an array of the values, ordered appropriately
 - Any exceptions or errors that come from your code should be semantic, capturable errors. For example, rather than a default error thrown by your language, your code should raise/throw a custom, semantic error that describes what went wrong in calling the methods you wrote for this lab
+- Added Breadth First Search
```

Comment:
Your whiteboard image isn't working for the first whiteboard anymore, going to need more information on Breadth First Search, the code for that and the whiteboard image.